### PR TITLE
Build diversity page

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Build careers diversity page
- Add test
- Redirect old diversity page to new diversity page
- Added item to navigation
- Remove old diversity pages

## QA

- Compare https://canonical-com-840.demos.haus/careers/company-culture/diversity with [design](https://www.figma.com/file/qCxGiBTgSFa21iVR7YsgWO/%2FCareers%2FDiversity?node-id=18-2374&t=2Naxc688KVcybL9u-0)
- Compare https://canonical-com-840.demos.haus/careers/company-culture/diversity with [copy docs](https://docs.google.com/document/d/1l1UGJToc0ErHUMN9W8hELvF5sDyVRO7N0xDzncqA2SQ/edit#heading=h.ov7lrxgwh2dv)
- https://canonical-com-840.demos.haus/careers/diversity should redirect to https://canonical-com-840.demos.haus/careers/company-culture/diversity 
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-179?atlOrigin=eyJpIjoiMTkwYjc4Y2M5NDkyNDc3MzlhMGZmZDI0MzM1NTZjODAiLCJwIjoiaiJ9

### CODE REVIEW

**Comment by marinacastejon** [comment]
Hi @carkod , 
I found a few discrepancies with the mock-up:
- The content before Accessibility awareness is missing
image

- The paragraph above Hanna's photo should not be aligned with her photo
image

- The sub-sections in "Behind the process" should have a smaller margin/padding according to the mock-ups:
image

- The suru background is not rendering like in the mock-up:
image

Discrepancies with the copy doc:
- CTA button label: see roles
- Hiring section has the wrong title, it's "Our hiring process"
- The copy is people-centric process is not correct

Thanks!

**Comment by camillet16** [comment]
Screen Shot 2023-04-20 at 12 58 49 PM

@carkod I played with the Suru.
```
background-position: bottom !important;
background-size: cover;
background-repeat: no-repeat;
```
Worked but the resize needs some adjustment on screen-resize.
Here's how @mtruj013 navigated it too if it helps [https://canonical-com-839.demos.haus/partners](url)

**Comment by camillet16** [comment]
I put a side-by-side comparison on Figma. I put the feedback there.

**Comment by carkod** [comment]
@marinacastejon @camillet16 sorry I think I just basically became a bit frustrated with Figma and didn't do a good job there.

**Comment by marinacastejon** [comment]
Hi @carkod the copy in the "people-centric process" section is not correct. Let me know when it's done and I'll review it again. Thanks

**Comment by marinacastejon** [comment]
@carkod as explained via MM please remove the section "People centric process" until confirmed by stakeholders, I left a comment in the [copy doc ](https://docs.google.com/document/d/1l1UGJToc0ErHUMN9W8hELvF5sDyVRO7N0xDzncqA2SQ/edit#)tagging you. Thank you

**Comment by carkod** [comment]
Given the situation (demos and deployments not working, IS already looking into that), I guess I can only get code and QA review @mtruj013 when you can?

**Comment by marinacastejon** [comment]
@carkod LGTM, thank you

**Comment by camillet16** [comment]
Screen Shot 2023-04-26 at 12 05 42 PM

- content for 'Fostering Participation' put class u-no-margin on the `<p> Working with a diverse community...`
- add padding-bottom: 1.5rem to the  'Fostering Participation' container.

<hr>

Screen Shot 2023-04-26 at 12 05 56 PM

- This specific text section has a padding-bottom: 1.5rem to create some space before Charlene's quote and photo.

<hr>

Screen Shot 2023-04-26 at 12 09 00 PM
- This specific text section also has a padding-bottom: 1.5 rem to create some space before the main content.

<hr>

Lyubo is handling the Suru. 😄  Thanks, Carlos!

File: templates/careers/company-culture/diversity.html [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+  </section>
+</div>
+
+<section class="p-strip--diversity u-no-padding--top">
+  <div class="u-fixed-width">
+    <hr class="u-no-margin--bottom u-hide--medium u-hide--small">
+  </div>
+  <div class="row">
+    <div class="col-medium-6 col-6">
+      <h3 class="p-muted-heading">Equal opportunities</h3>
```
Comment (mtruj013) [inline templates/careers/company-culture/diversity.html:33]: This heading isn't on the doc

File: templates/careers/company-culture/diversity.html [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+  </div>
+</div>
+
+<div class="p-strip--diversity">
+  <div class="u-fixed-width">
+    <hr class="u-no-margin--bottom">
+  </div>
+  <div class="row">
+    <div class="col-3 col-medium-2">
+      <h3 class="p-muted-heading">Behind the process</h3>
```
Comment (mtruj013) [inline templates/careers/company-culture/diversity.html:211]: Heading not on the doc

File: templates/careers/company-culture/diversity.html [hunk 3]
```diff
@@ -0,0 +1,263 @@
+{% extends '/careers/base-tabs.html' %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1PAeP_ZO4JzSBCi-fRp6F7SYtSSRY-4DcWLUf9HJaMxs/edit{% endblock meta_copydoc %}
```
Comment (mtruj013) [inline templates/careers/company-culture/diversity.html:2]:
```suggestion
{% block meta_copydoc %}https://docs.google.com/document/d/1l1UGJToc0ErHUMN9W8hELvF5sDyVRO7N0xDzncqA2SQ/edit#{% endblock meta_copydoc %}
```

**Review by mtruj013 — APPROVED** [review]
LGTM, just some minor comments in the code. There are a couple of headings that are missing from the doc, I assume it's just a matter of updating it if everyone's happy to have them? Also, could we put the template for company culture in the company culture folder? Bit of a hard find when the company culture index and child pages (diversity) are separated like that I think

**Comment by carkod** [comment]
> 
> * content for 'Fostering Participation' put class u-no-margin on the `<p> Working with a diverse community...`
> * add padding-bottom: 1.5rem to the  'Fostering Participation' container.
> 
> * This specific text section has a padding-bottom: 1.5rem to create some space before Charlene's quote and photo.
> 
> - This specific text section also has a padding-bottom: 1.5 rem to create some space before the main content.
> Lyubo is handling the Suru. smile Thanks, Carlos!

@camillet16 can you take another look when you can?

**Pull request merged**